### PR TITLE
Feat/issue 57 migrate videojs vast vpaid

### PR DESCRIPTION
**Pull request by carpasse**

### CODE REVIEW

File: packages/video-ad-sdk/src/adUnit/helpers/dom/elementObservers/onElementResize.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable promise/prefer-await-to-callbacks */
-import debounce from 'lodash/debounce';
+import debounce from 'lodash.debounce';
```
Comment (Fetz) [inline packages/video-ad-sdk/src/adUnit/helpers/dom/elementObservers/onElementResize.js]: if we start adding more modules of lodash, then importing the normal lodash is better if we use lodash-webpack-plugin

Comment (carpasse) [reply packages/video-ad-sdk/src/adUnit/helpers/dom/elementObservers/onElementResize.js]: I don't really plan on adding more to be honest.

**Comment by Fetz** [comment]
What is the page you want to be in the github page? The docs or the static demo page?

Because currently, the docs is the one being used by the github page see https://github.com/MailOnline/mol-video-ad-sdk/blob/next-release/.travis.yml#L18 for more details

**Comment by carpasse** [comment]
For now the docz. The static page is just to ensure everything works. We need to build a proper demo asap

**Pull request merged**